guides/common/attributes-titles.adoc

```diff
@@ -24,6 +24,7 @@
 :MonitoringDocTitle: Monitoring {Project} performance
 :PlanningDocTitle: Planning for {ProjectName}
 :ProvisioningDocTitle: Provisioning hosts
+:ProvisioningInstallerDocTitle: Provisioning hosts by using installer automation
```

Comment:
I'm not sure what "installer automation" means. In the first paragraph, I see "you can define and automate fine-grained provisioning for a large number of hosts". Are you saying that this is a guide for automated provisioning? 

A user might think this guide is about automating an 'installer' that then provisions the hosts. That implies that there is a guide for provisioning hosts by manually using an installer. Do you see what I mean? I think the title is a little murky. 

The titles do not need to start with "Provisioning hosts ..." if there is a good reason not to do so. If this guide is for automated provisioning, perhaps call it something like "Automated host provisioning". Apologies in advance if I am missing something obvious. I have not read the entire guide.

Reply:
All three guides are about automated or semi-automated provisioning, so "Automated host provisioning" for one of them is not the best idea.
In the context of RHEL, there are multiple installation guides - for interactive installation and automatic.
![image](https://github.com/user-attachments/assets/103129ca-7919-4f0a-bb58-f59afcdda39b)

I was trying to express that what Foreman does is not the interactive installation but the (semi-)automatic installation, where Anaconda (the installer) is automated by means of the Kickstart configuration files that Foreman provides. An analogic way is used for installing other operating systems as well.

One wouldn't use Foreman to install the OS interactively, so we could leave out "automated" from the title and go with "Provisioning hosts by using an installer" because our particular guide is about provisioning methods that are installer based as opposed to provisioning methods that are image based (the other 2 guides).
If we want, we can have a title for Satellite that is more specific, ie. "Provisioning hosts by using Anaconda".

I can't think of any other titles that would be a better fit. Any other ideas, @apinnick ?

Reply:
If no one else objects to "Provisioning hosts by using Anaconda", that sounds good. My only concern is that it might be too narrow. I don't know the subject well enough.

Reply:
Well, the only feature that is "out of place" is the Discovery chapter. And since it's quite long, I keep wondering if we should split it off to another guide. What do you think?

The "Preparing" chapters are all prerequisites for network boot and boot disks and I don't think we want to make the title any longer. We can be more specific in the subtitle for Satellite.